src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -136,7 +134,7 @@ public KafkaProducerDiagnostic getDiagnostic() {
 class MaxwellKafkaProducerWorker extends AbstractAsyncProducer implements Runnable, StoppableTask {
 	static final Logger LOGGER = LoggerFactory.getLogger(MaxwellKafkaProducer.class);
 
-	private final KafkaProducer<String, String> kafka;
+	private final Producer<String, String> kafka;
```

Comment:
this change... are we sure this change is cool with older versions of kafka libraries?

Reply:
It's just an interface for the class, it's been there since 0.8 so I assume it's fine: https://kafka.apache.org/082/javadoc/index.html?org/apache/kafka/clients/producer/KafkaProducer.html